reference/apcu/functions/apcu-key-info.xml

```
@@ -38,7 +38,7 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   null or array info
+   An array containing the detailed information about the cache key, or &null; on failure.
```

```suggestion
   An array containing the detailed information about the cache key, or &null; if the key does not exist.
```
Maybe?

---

Maybe both? While a non existing key is likely the cause for `null` return, `null` is also returned if locking of the cache fails: https://github.com/krakjoe/apcu/blob/b68953ccbc1a5234c2caf8ed5800afd222270948/apc_cache.c#L1145-L1147

---

Locking can't really fail, it's just implemented defensively. Possibly it should just throw a fatal error instead.

---

Ah, okay, than I think we're good here. :)